Mathlib/ModelTheory/Syntax.lean

```diff
@@ -742,21 +742,18 @@ protected nonrec abbrev not (φ : L.Formula α) : L.Formula α :=
 protected abbrev imp : L.Formula α → L.Formula α → L.Formula α :=
   BoundedFormula.imp
 
-/-- Given a map `f : α → β ⊕ γ`, `iAlls f φ` transforms a `L.Formula α`
-into a `L.Formula β` by renaming variables with the map `f` and then universally
+variable (β)
```

Comment:
Do you want to switch back to `{}` after `iExs`? Or was it your intention to keep `()`?
Maybe `variable in` is more appropriate?

Reply:
That was a mistake. Well spotted. It's now fixed.